libpod/networking_common.go

```diff
@@ -632,14 +640,22 @@ func (r *Runtime) ConnectContainerToNetwork(nameOrID, netName string, netOpts ty
 	return ctr.NetworkConnect(nameOrID, netName, netOpts)
 }
 
-// normalizeNetworkName takes a network name, a partial or a full network ID and returns the network name.
+// normalizeNetworkName takes a network name, a partial or a full network ID and
+// returns the network name and network_interface name for macvlan and ipvlan
```

Comment:
This isn't just used for macvlan and ipvlan, though? It's used for all drivers.

Reply:
The comment about the drivers is for the latter part. That is, network_interface name. Do you want me to change that?

Reply:
Updated.